### DISCUSSION TITLE
Validate user_meta before registration

### DISCUSSION THREAD

**Discussion by matveycodes** [nicumicle/simple-jwt-login#20] [Q&A]
How can I validate `user_meta` object before registering a user? For example, I would like to validate `gender` field so that it contains either `male` or `female`. How can I do that?

**Comment by nicumicle** [comment]
Hello @matveycodes,

I can add a hook before registering the user. This way, you will be able to validate the meta. I will come back with updates once I release the new version.

Best regards,
Nicu.
